### DISCUSSION TITLE
ZbSend RGB color

### DISCUSSION THREAD

**Discussion by pingel20** [arendst/Tasmota#10440] [Q&A] [answered]
Is it possible to send an RGB color using Zigbee (ZbBridge)?

Like
ZbSend { "Device":"0x4773", "Color":"#F433B9" }

**Comment by s-hadinger** [comment, marked as answer]
No. Zigbee standard does not know RGB format. You need to convert to Hue/Sat or X/Y depending on what the bulb supports.

**Comment by pingel20** [comment]
Thank you.
Almost all color pickers output an RGB value, that's why I asked.

I will go and look for a conversion algorithm.

**Comment by s-hadinger** [comment]
True. However this need sounds reasonable, I could integrate it into Zigbee2Tasmota and automatically output Hue/Sat and X/Y. The only caveat is that it actually needs 2 zigbee packets, one for color and one for brightness. But nothing impossible though :)

On my todo list.
